h2o-hadoop-2/h2o-mapreduce-2-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1821,6 +1830,8 @@ private JobWrapper submitJob(Configuration conf) throws Exception {
     j.setOutputKeyClass(Text.class);
     j.setOutputValueClass(Text.class);
 
+    new HiveTokenGenerator(driverDebug).addHiveDelegationToken(j, hiveHost, hivePrincipal);
```

Comment:
I would probably leave it on - at least for now. Otherwise, it doesn't generate any output.

Reply:
it can be enabled by -driver_debug so we dont need to have it always on